libr/meson.build

```diff
@@ -109,6 +109,9 @@ if not no_user_plugins
     anal_plugins += [ 'riscv' ]
     parse_plugins += [ 'riscv_pseudo' ]
   endif
+  if user_plugins.contains('loongarch')
```

Comment:
i think it's better to include it under `use_gpl` too

Reply:
ok！
and “asm.arch: cannot find (loongarch)” how do I handle it, is there a reference code?

Reply:
let me merge and try the error you mention.